gateway/src/config/persistence/paths.rs

```diff
@@ -39,6 +40,10 @@ pub struct GatewayPaths {
     #[serde(alias = "persistent_storage")]
     pub clients_storage: PathBuf,
 
+    /// Path to sqlite database containing all persistent stats data.
+    #[serde(alias = "persistent_stats_storage")]
```

Comment:
if it's a brand new field, what why do we need the alias? (it existed on `clients_storage` because we renamed it : ) )

Reply:
You're right, we don't need it
Removed in 6486609